packages/grafana-ui/src/components/Sparkline/Sparkline.tsx

```diff
@@ -0,0 +1,166 @@
+import React, { PureComponent } from 'react';
+import {
+  compareDataFrameStructures,
+  DefaultTimeZone,
+  FieldSparkline,
+  IndexVector,
+  DataFrame,
+  FieldType,
+  getFieldColorModeForField,
+  FieldConfig,
+} from '@grafana/data';
+import { AxisPlacement, DrawStyle, GraphFieldConfig, PointVisibility } from '../uPlot/config';
+import { UPlotConfigBuilder } from '../uPlot/config/UPlotConfigBuilder';
+import { UPlotChart } from '../uPlot/Plot';
+import { Themeable } from '../../types';
+
+export interface Props extends Themeable {
+  width: number;
+  height: number;
+  config?: FieldConfig<GraphFieldConfig>;
+  sparkline: FieldSparkline;
+}
+
+interface State {
+  data: DataFrame;
+  configBuilder: UPlotConfigBuilder;
+}
+
+const defaultConfig: GraphFieldConfig = {
+  drawStyle: DrawStyle.Line,
+  showPoints: PointVisibility.Auto,
+  axisPlacement: AxisPlacement.Hidden,
+};
+
+export class Sparkline extends PureComponent<Props, State> {
```

Comment:
This class is similar to GraphNG -- but it knows that the input is only 1 or 2 fields

Reply:
Great